stm32cube/stm32f7xx/README

```diff
@@ -59,4 +59,10 @@ Patch List:
     Impacted files:
      drivers/include/Legacy/stm32_hal_legacy.h
 
+   *Fix Use the correct register DESCO
+    This fix ensures that the TX timestamp is properly set by correctly
+    checking the ETH_DMATXDESC_LS and ETH_DMATXDESC_TTSS flags in the DESCO register.
+    Impacted files:
+     stm32cube/stm32f7xx/drivers/src/stm32f7xx_hal_eth.c
+
```

Comment:
Could add a reference to bugtracker ticket:
either `https://github.com/STMicroelectronics/stm32f7xx_hal_driver/pull/9` or `ST Internal Reference: 185461`